### ISSUE TITLE
How was this created? There are over 3700 signals

### ISSUE THREAD

**Issue by larryqiann** [joshwardell/model3dbc#21]
Hi,

First of all thank you for providing this extremely valuable resource, this is hours of fun once I receive my diagnostic cables!!!

Sorry if this is off topic or not the best place to ask, but after going through the whole list, I am surprised at how many messages and signals are present - surely this would be a gargantuan effort to reverse engineer and create over 3,500 definitions manually. 

What kind of a process was used to create this kind of file, if it is possible to share any insight? Was there a particular source for each of the parameter names/were these extracted semi-automatically from a rooted MCU?

Thanks!!!

**Comment by DanubeRS** [comment]
My guess is it came from the Toolbox v2 software, and its associated plugins.

Since then, Tesla has moved to a web-based diagnostic platform, and I don't know if the signals are recoverable from that suite.

**Comment by icepick3000** [comment]
> My guess is it came from the Toolbox v2 software, and its associated plugins.
> 
> Since then, Tesla has moved to a web-based diagnostic platform, and I don't know if the signals are recoverable from that suite.

Is this something i can access as an individual or is this only available to the Tesla service centers?

**Comment by DanubeRS** [comment]
Yes, although it can be costly depending on how long you purchase the license for.

**Comment by icepick3000** [comment]
Do you have a link for me?
